specs/guidelines.md

```diff
@@ -89,18 +89,29 @@ specially on IoT networks which deliver data only at specific timeslots.
 ## Internationalization (i18N)
 
 There can be certain entity attributes which content is subject to be internationalized. For instance, the description of a Point of Interest.
-The i18N guidelines for the FIWARE Data Models are as follows:
+The internationalization (i18N) guidelines for the FIWARE Data Models are defined as follows:
 
-+ By default the value of an attribute subject to be internationalized shall be expressed in *British English* (`en-GB`).
-+ For each language variant available for an attribute subject to be internationalized there shall be an additional
++ By default, the value of an attribute subject to be internationalized shall be expressed in **American English** (`en-US`).
```

Comment:
I'm not so sure about **shall** its a bit too prescriptive - should the default **always** be in English?  Is this necessarily the case? 

Take the city of *Livorno* in Italy. There is an obscure historical English exonym *Leghorn* 

If we insist on English as  a default then we would get:

```json
{
    "name": "Leghorn",
    "name_it": "Livorno"
}
```

Assuming there is no exonym in Spanish,  a Spanish reader would get an English geography lesson for the place name.  The default should be:

> the common international name in latin script, usually expressed in **American English**

Hence the following would be correct:

### English has a common exonym

```json
{
    "name": "Brussels",
    "name_fr": "Bruxelles",
    "name_nl": "Brussel",
    "name_de" : "Brüssel"
}
```
### English has a uncommon exonym
```json
{
    "name": "Livorno",
    "name_en": "Leghorn"
}
```
### Romanization of Chinese Script 
```json
{
    "name": "Beijing",
    "name_zh": "北京"
}
```

### No English Exonym - use common name
```json
{
    "name": "Tallinn",
    "name_de": "Reval"
}
```

Reply:
good points one more time. I will try to clarify